package/kernel/linux/modules/crypto.mk

```diff
@@ -839,3 +841,121 @@ endef
 
 $(eval $(call KernelPackage,crypto-xts))
 
+define KernelPackage/crypto-curve25519
```

Comment:
Please consider sequence.

Reply:
What do you mean by sequence?

Reply:
e.g. `crypto-curve25519` should be put after `crypto-cts`.

Reply:
> e.g. crypto-curve25519 should be put after crypto-cts.

So, "alphabetical ordering"?

Reply:
Yeah I'll make the change. @dengqf6 do you want to coordinate mt7621 roll-out somehow - I can check your branch against my er-x. You can ping me on IRC if you want.

Reply:
> So, "alphabetical ordering"?

Oh yes, sorry for my poor Eng.

Reply:
https://github.com/openwrt/openwrt/pull/3885